requirements.txt

```diff
@@ -2,5 +2,5 @@ markdown
 -e git://github.com/edx/xblock-utils.git#egg=xblock-utils
 ddt
 mock
-django-nose==1.4.6
+django-nose
```

Comment:
I'm not sure why these are repeated in requirements and requirements-dev, but please don't unpin, if already pinned.